platform/app/prisma/migrations/20260809120000_process_manager_retention_indexes/migration.sql

```
@@ -0,0 +1,55 @@
+-- of the build, which blocks INSERT while it runs. That is why the one-time
+-- purge in dev/docs/runbooks/process-manager-table-purge.md runs FIRST: over a
+-- near-empty heap these builds are near-instant, whereas over the pre-purge
+-- 2.8M inbox rows the same build would hold the lock long enough to back up
+-- automation ingestion. If this migration is ever replayed against a large
+-- table, use CREATE INDEX CONCURRENTLY by hand instead, which cannot run
+-- inside the migration transaction.
+
+-- CreateIndex
+CREATE INDEX "ProcessManagerOutbox_dispatchedAt_idx"
+  ON "ProcessManagerOutbox"("dispatchedAt")
+  WHERE "status" = 'dispatched';
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)